web/components/elements/DropdownSearch.module.scss

```diff
@@ -0,0 +1,39 @@
+.dropdownItem {
+    padding: 0.25em 0.5em;
+    border-radius: 8px;
+}
+
+.dropdownItem:hover {
+    cursor: pointer;
+    background-color: rgba(54, 54, 54, 0.138);
+}
```

Comment:
Let me tell you about the magic and wonders of nested SCSS.
Also, `cursor` only affects the cursor on hover anyway.

```suggestion
    cursor: pointer;

    &:hover {
        background-color: rgba(54, 54, 54, 0.138);
    }
}
```

Reply:
> Also, cursor only affects the cursor on hover anyway.

I am so dumb :D